lib/cmd/fh3/env/create.js

```diff
@@ -9,31 +9,39 @@ module.exports = {
     [{
       cmd : 'fhc env create --app=<app> --name=<name> --value=<value> --env=<environment>',
       desc : i18n._("Create a cloud app environment variables for the <app> in the <env> with the <name> and <value>")
+    },{
+      cmd : 'fhc env create --app=<app> --name=<name> --value=<value> --env=<environment> --mask',
+      desc : i18n._("Create a cloud app environment variables for the <app> in the <env> with the <name> and <value> and masked")
     }],
   'demand' : ['app', 'name'],
   'alias' : {
     'app': 'a',
     'name': 'n',
     'value': 'v',
     'env': 'e',
+    'mask': 'm',
     'json': 'j',
     0: 'app',
     1: 'name',
     2: 'value',
-    3: 'env',
-    4: 'json'
```

Comment:
The json flag was not removed. Since it is a modifier parameter it should not be by position. 
The user can use it any position of the command by --json or --j